dev/com.ibm.ws.messaging.common/resources/com/ibm/ws/sib/processor/CWSIPMessages.nlsprops

```diff
@@ -1154,6 +1161,12 @@ ANYCAST_STREAM_UNAVAILABLE_CWSIP0481=CWSIP0481E: A stream to communicate with th
 ANYCAST_STREAM_UNAVAILABLE_CWSIP0481.explanation=A stream to communicate with the localisation messaging engine for the destination is not available. This may be because the stream is still being initialized, or due to a network partition.
 ANYCAST_STREAM_UNAVAILABLE_CWSIP0481.useraction=Verify that it is possible to communicate with the localisation messaging engine, and then try again.
 
+# -------------------------------------------------------------------------------------------------
+# {0} The ProtocolItemStream object
+MISSING_COMPLETED_PREFIX_IN_STREAM_CWSIP0511=CWSIP0511W: A malformed ProtocolItemStream {0} has been found. The creation of the stream will be deferred so that it may be properly formed.
```

Comment:
change
has been found
to
was found

change
will be deferred
to 
is deferred

change
may be properly formed.
to
can be formed properly.

Reply:
Agreed